### PR TITLE
Add aptX libraries

### PR DESCRIPTION
**Pull request by mthomos**

### CODE REVIEW

File: s2/s2-vendor.mk [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # This file is generated by device/leeco/s2/setup-makefiles.sh
 
 PRODUCT_COPY_FILES += \
+    vendor/leeco/s2/proprietary/vendor/lib/libaptX-1.0.0-rel-Android21-ARMv7A.so:system/vendor/lib/libaptX-1.0.0-rel-Android21-ARMv7A.so \
```
Comment (harryyoud) [inline s2/s2-vendor.mk:18]: Wrong ordering. You clearly haven't generated these makefiles, you've modified them by hand

**Comment by harryyoud** [comment]
This device also has no aptX certification from its OEM, so shipping aptX is strictly forbidden

**Comment by mthomos** [comment]
Yes modified by hand. I have thought to insert these aptX libraries beacuse i have tested aptX and it worked and it would be nice to introduce aptX support to LOS

**Comment by jrior001** [comment]
The 8996 leecos don't (to my knowledge) have the certified actions to run these on stock and therefore will  not accept them on Lineage either. Unfortunately the risk a DCMA request outways the minimal benefits of shipping these blobs.

**Comment by jrior001** [comment]
opps just realized this was for S2. Same logic applies though. If stock is not certified for aptX we will not ship it either.

**Pull request closed without merging**